core/domain/opportunity_services_test.py

```diff
@@ -185,6 +185,38 @@ def test_new_opportunity_with_adding_exploration_in_story_node(self):
         self.assertEqual(opportunity.topic_name, 'topic')
         self.assertEqual(opportunity.story_title, 'A story')
 
+    def test_get_translation_opportunities_with_no_translations_in_review(
```

Comment:
Shouldn't this test name be swapped with the one below?

Reply:
Done